website/package.json

```diff
@@ -13,8 +13,9 @@
   },
   "dependencies": {
     "@headlessui/react": "^1.6.6",
-    "@heroicons/react": "^1.0.6",
-    "@markdoc/markdoc": "^0.1.2",
+    "@heroicons/react": "^2.0.0",
```

Comment:
I was thinking about doing this upgrade from dependabot changes today, glad to see it rolled in here